core/executor/src/wasm_executor.rs

```diff
@@ -198,6 +198,17 @@ impl_function_executor!(this: FunctionExecutor<'e, E>,
 		debug_trace!(target: "sr-io", "free {}", addr);
 		Ok(())
 	},
+	ext_set_changes_trie_config(block: u64, digest_interval: u64, digest_levels: u32) => {
```

Comment:
any reason this wasn't done as a special storage item like `:code` and `:heappages`?

Reply:
No specific reasons for this, but 'system' storage items has already caused some troubles before (like: prefix removal was required outside of runtime in storage-with-prefix PR, panic if the entry does not exists, ...). Also we need to announce block number anyway (this probably will go away with pruning PR, but still required for now).

That said, I could replace this with `ext_set_block_number` (which again is probably a 
temporary decision until pruning PR).